File: Programmers/코딩테스트 입문/Python/Lv.0/[Lv.0] 피자 나눠 먹기 (2)/[Lv.0] 피자 나눠 먹기 (2)_00.py
def solution(n):
    # 피자의 조각 수 = 피자의 판 수 * 6
    # n으로 나누기
    # 나머지가 있으면 피자의 판 수 증가
    for i in range(1, n + 1):
        if 6 * i % n == 0:
            return i

File: Programmers/코딩테스트 입문/Java/Lv.0/[Lv.0] 피자 나눠 먹기 (2)/[Lv.0] 피자 나눠 먹기 (2)_00.java
class Solution {
    public int solution(int n) {
        int answer = 0;

        // 피자의 판 수 = i
        // 최소 1판이 필요하므로 1부터 시작
        for (int i = 1; i <= n; i++) {
            // 피자의 조각 수 = i * 6
            // 피자의 조각 수를 n으로 나눴을 때,
            // 나머지가 0일 경우 i 반환, 아닐 경우 i 증가
            // 1 * 6 % 6 = 0
            if (i * 6 % n == 0) {
                answer = i; // 1
                break;
            }
        }

        return answer;
    }
}
